.github/workflows/release_community_cromwell.yml

```diff
@@ -0,0 +1,99 @@
+name: Release Community Cromwell
+
+on:
+  workflow_dispatch: 
```

Comment:
Nit, but do you want to set up a manual trigger to be able to run this workflow on any branch in case you need to further debug once this has been merged? Something like this should do it: https://github.com/DataBiosphere/terra-docker/blob/a0cf5769e6b469ba5813c810c119b10f26915422/.github/workflows/publish.yml#L2

Reply:
I do, but it looks like I get that for free - this is what I see when I run the action in my fork: 
<img width="352" alt="Screenshot 2025-02-19 at 9 33 41 AM" src="https://github.com/user-attachments/assets/8328c9bc-e6c7-4474-b29d-42e49f9907ae" />
 Maybe I'm misunderstanding though, I haven't actually tried with a different branch.

Reply:
I guess you only care about the workflow file, and if that's the case you should be all set. In the example I shared, the branch input is to chose the code from your PR branch, in addition to selecting the workflow from that branch too
<img width="503" alt="Screenshot 2025-02-19 at 9 59 38 AM" src="https://github.com/user-attachments/assets/cdb056a5-9b36-4a47-bde4-4111e79109cf" />

Reply:
Ah, got it - for this specific workflow I think we do always want to use `develop` and `master`, and use a fork if we want to test without creating real releases. Feels like a footgun to give users a dropdown to create a real release from an arbitrary branch.